Dockerfile

```diff
@@ -1,28 +1,32 @@
-FROM node:10-alpine
+FROM node:10-alpine as base
 LABEL Description="Web Frontend of the Social Network Human-Connection.org" Vendor="Human-Connection gGmbH" Version="0.0.1" Maintainer="Human-Connection gGmbH (developer@human-connection.org)"
 
 # Expose the app port
-EXPOSE 3000
-
+ARG BUILD_COMMIT
+ENV BUILD_COMMIT=$BUILD_COMMIT
 ARG WORKDIR=/nitro-web
 RUN mkdir -p $WORKDIR
 WORKDIR $WORKDIR
 
 # See: https://github.com/nodejs/docker-node/pull/367#issuecomment-430807898
 RUN apk --no-cache add git
 
-# Install Web Application
-COPY package.json .
-COPY yarn.lock .
-COPY styleguide/ ./styleguide
-RUN yarn install --production=false --frozen-lockfile --non-interactive
+COPY . .
 
-# Install and build Styleguide
-COPY styleguide/ ./styleguide
+FROM base as build-and-test
+RUN yarn install --production=false --frozen-lockfile --non-interactive
 RUN cd styleguide && yarn install --production=false --frozen-lockfile --non-interactive \
     && cd .. \
     && yarn run styleguide:build
-
 COPY . .
-RUN ["yarn", "run", "build"]
+RUN yarn run build
+
+FROM base as production
+ENV NODE_ENV=production
+COPY --from=build-and-test ./nitro-web/node_modules ./node_modules
```

Comment:
@mattwr18 can you try to compress the docker image and `yarn install` the dependencies instead of copying them from `build-and-test`?

Reply:
what do you mean compress the docker image? you had suggested the approach of running `yarn install` in the production image to avoid copying over the `devDependencies` earlier, but it doesn't pass the build because of this dependency `nuxt-sass-resources-loader`, maybe it is incorrectly in `devDependencies`?

Reply:
From their [github page](https://github.com/anteriovieira/nuxt-sass-resources-loader): `"Note that installing as a dev dependency --save-dev or -D will not work correctly."` 
So yes It can`t be only in the dev devDepemdencies!